src/modules/ui/components/layout/index.js

```diff
@@ -108,6 +115,13 @@ class Layout extends React.Component {
             resizerChildren={downPanelInRight ? vsplit : hsplit}
             onDragStarted={onDragStart}
             onDragFinished={onDragEnd}
+            onChange={
```

Comment:
Oh and also this, check out the conflict that came up due to another PR getting merged, make sure to it plays along :)